components/automate-ui/src/app/pages/project/details/project-details.component.html

```diff
@@ -1,4 +1,5 @@
 <app-settings-sidebar></app-settings-sidebar>
+<chef-loading-spinner *ngIf="isLoading" size='50' fixed></chef-loading-spinner>
```

Comment:
this spinner presents while the current project's details and project rules load